app/experimenter/nimbus-ui/src/components/PageRequestReview/DraftStatusOperations.tsx

```diff
@@ -100,15 +102,23 @@ export const DraftStatusOperations = ({
     /* istanbul ignore next until EXP-1055 & EXP-1062 done */
     case DraftStatusOperationsState.LaunchApproveOrReject:
       return (
-        <FormApproveOrRejectLaunch
-          {...{
-            isLoading,
-            launchRequestedByUsername,
-            onApprove: handleLaunchApprovalClicked,
-            onReject: () =>
-              setDraftUIState(DraftStatusOperationsState.LaunchRejection),
-          }}
-        />
+        <>
+          {rsRequestTimedOut && (
+            <p className="border border-danger text-danger px-2 py-4 mb-3">
```

Comment:
Small nit: This looks a bit inconsistent with [the launch pending alert over here](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/components/PageRequestReview/DraftStatusOperations.tsx#L174-L183) - might want to use an Alert here similar to that?